components/admin_console/admin_console.tsx

```diff
@@ -10,6 +10,7 @@ import {Role} from 'mattermost-redux/types/roles';
 import {ConsoleAccess} from 'mattermost-redux/types/admin';
 import {CloudState} from 'mattermost-redux/types/cloud';
 import {Team} from 'mattermost-redux/types/teams';
+import {DeepPartial} from 'mattermost-redux/types/utilities';
```

Comment:
The way we had originally defined `DeepPartial`, it was globally available without being imported. That doesn't work when we're building a package to be shipped externally, so now this needs to be imported wherever it's used.

This should also fix one of the issues that people using the web app directly have been facing since I started moving stuff around